doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -1235,6 +1235,11 @@ Other libraries
 
   * Added the :c:func:`date_time_now_local` function to the API.
 
+* :ref:`dk_buttons_and_leds_readme` library:
+
+  * Added experimental no interrupts mode for button handling, which can be enabled with the :kconfig:option:`CONFIG_DK_LIBRARY_BUTTON_NO_ISR` Kconfig option.
+    It is a workaround for the nRF54L15 PDK revision 0.2.X, where Button 3 and 4 are connected to the GPIO port which does not support interrupts.
```

Comment:
```suggestion
  * Added an experimental no interrupts mode for button handling. 
    You can enable it with the :kconfig:option:`CONFIG_DK_LIBRARY_BUTTON_NO_ISR` Kconfig option.
    It is a workaround that can be used to avoid using the GPIO interrupts, however, it increases the consumption.
```